src/js/module.js

```diff
@@ -28,6 +28,7 @@ module.exports = iotjs_module_t;
 
 
 iotjs_module_t.cache = {};
+iotjs_module_t.remoteCache = {};
```

Comment:
Do we need a separate remote cache? Can't we just put the sources into the normal cache?

Reply:
The only reason there is a separate cache is for better visibility.
Since there is a separate function for loading them as well, I thought it would clearer to read. But that's just my opinion.